src/Microsoft.Extensions.ProcessHelper.Sources/project.json

```diff
@@ -0,0 +1,17 @@
+﻿{
+  "version": "1.0.0-*",
+  "shared": "*.cs",
+  "dependencies": {
+  },
+  "frameworks": {
+    "net451": { },
+    "netstandard1.4": {
+      "dependencies": {
+        "System.Diagnostics.Process": "4.1.0-*"
```

Comment:
I thought you don't need *any* of this in a .Sources package... when I made one the other day, nothing here mattered as long as there was one TFM that was compatible everywhere, and that would be `netstandard1.0` with no dependencies at all.